public/dist/js/restaurant-week.app.js

```diff
@@ -14,12 +14,13 @@
 	var constants = {
 		urls: {
 			templates: {
-				restaurant: '/_TEST/_templates/restaurant/restaurant.template.html',
-				restaurantList: '/_TEST/_templates/restaurant-list/restaurant-list.template.html'
+				restaurant: '//staging.baltimorecountymd.gov/_Restaurant Week/app/restaurant.template.html',
+				restaurantList: '//staging.baltimorecountymd.gov/_Restaurant Week/app/restaurant-list.template.html'
```

Comment:
GitHub's changes are confusing me a little, because it's showing that this changed from `_TEST` to `staging` even though the original problem was that it changed from `staging` to `_TEST`.  I'm viewing changes from all commits.  I'm just going to dismiss this as noise.